collection-scripts/gather_core_dumps

```diff
@@ -4,10 +4,35 @@ CORE_DUMP_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_core_dumps"}
 
 mkdir -p ${CORE_DUMP_PATH}/
 
+function get_dump_off_node { 
+    #Start Debug pod force it to stay up for 60 seconds, supress Stdout 
+    oc debug node/$1 -- /bin/bash -c 'sleep 60' > /dev/null 2>&1 &
+    
+    #Clear local variables before assignment
+    local n=0
+    local debug=""
+
+    # Make sure pod is avaliable exit after 5 seconds if not 
+    while [[ -z "$debug"  && $n -le 10 ]]
+    do    
+        #Get name of Debug Pod for relevant node 
+        debug=$(oc get pods --all-namespaces | grep $1-debug | awk -F' ' '{print $2}')
```

Comment:
won't the debug pod be in the current namespace?   line 9 should be in the current namespace.

Reply:
Yep it should always pop up in the current namespace, I can remove the `--all-namespaces` flag